qemu/tests/cfg/hugepage_reset.cfg

```diff
@@ -0,0 +1,17 @@
+- hugepage_reset:
+    type = hugepage_reset
+    start_vm = no
+    pre_command = 'echo 3 > /proc/sys/vm/drop_caches && echo 1 > /proc/sys/vm/compact_memory'
+    mem = 4096
+    origin_nr = 8
+    # Please set hugepage in kernel command line before this test:
```

Comment:
@maxujun Yes you need to prepare the environment before the test, please check this comment, thanks.

Reply:
[root@ibm-p9b-24 ~]# dmesg |grep huge
[    0.000000] Kernel command line: root=/dev/mapper/rhel_ibm--p9b--24-root ro crashkernel=auto rd.lvm.lv=rhel_ibm-p9b-24/root rd.lvm.lv=rhel_ibm-p9b-24/swap  default_hugepagesz=2G hugepagesz=2G hugepages=8
[    0.000000] HugeTLB: unsupported default_hugepagesz=2G
[    0.000000] HugeTLB: unsupported hugepagesz=2G
@MiriamDeng Could you help confirm?